### PR TITLE
libatomic_ops

### PR DESCRIPTION
**Pull request by bramtayl**
I know this will fail, this is just to serve as an example for an issue I will raise shortly

### CODE REVIEW

File: L/libatomic_ops/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libatomic_ops"
+version = v"7.6.10"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/ivmai/libatomic_ops/releases/download/v7.6.10/libatomic_ops-7.6.10.tar.gz")
```
Comment (staticfloat) [inline L/libatomic_ops/build_tarballs.jl:10]: This is missing a hash

File: L/libatomic_ops/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
```
Comment (staticfloat) [inline L/libatomic_ops/build_tarballs.jl:27]: This needs to be `Product[]` not just `[]`, since it has no members.

**Comment by bramtayl** [comment]
It's listed as a build requirement for PulseAudio: https://www.freedesktop.org/wiki/Software/PulseAudio/Download/ . If I make the changes above, would I be able to use when building PulseAudio_jll?

**Comment by staticfloat** [comment]
Yeah, should be.

**Pull request closed without merging**